src/plugins/formvalid/formvalid-en.hbs

```diff
@@ -759,12 +676,10 @@
 				&lt;input type="checkbox" name="example6" value="4" id="tech8" /&gt;
 				&lt;label for="tech8"&gt;Fusion reactor&lt;/label&gt;
 			&lt;/div&gt;
-		&lt;/fieldset&gt;
-					</code>
-				</pre>
+		&lt;/fieldset&gt;</code></pre>
 			</details>
 
-			<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset" class="btn btn-default" />
+			<input type="submit" value="Submit" class="btn btn-primary" /> <input type="reset" value="Reset page to defaults" class="btn btn-link btn-sm show p-0 pt-4" />
```

Comment:
@duboisp @delisma Was this reset button intentionally meant to resemble the older reset button design?